cluster/kube-util.sh

```diff
@@ -52,7 +52,11 @@ function set-federation-zone-vars {
   elif [[ "$KUBERNETES_PROVIDER" == "gke"  ]];then
 
     export CLUSTER_NAME="${USER}-${zone}"
+    export ZONE="${zone}"
+    export KUBE_GKE_NETWORK="${CLUSTER_NAME}-network"
+    export OVERRIDE_CONTEXT="gke_${PROJECT}_${ZONE}_${CLUSTER_NAME}"
 
+    source "${KUBE_ROOT}/cluster/gke/util.sh"
```

Comment:
This is the same deal as with `cluster/gce/util.sh` - The provider utils must be re-sourced so that computed values get refreshed when the zone context is changed.